src/utils/logging.rs

```diff
@@ -18,33 +18,31 @@ pub fn debug_log_to_file(message: String) -> io::Result<()> {
         .create(true)
         .open(MOSAIC_TMP_LOG_FILE)?;
     file.write_all(message.as_bytes())?;
-    file.write_all("\n".as_bytes())?;
-
-    Ok(())
+    file.write_all(b"\n")
 }
 
 pub fn debug_log_to_file_pid_0(message: String, pid: RawFd) -> io::Result<()> {
     if pid == 0 {
-        debug_log_to_file(message)?;
+        debug_log_to_file(message)
+    } else {
+        Ok(())
     }
-
-    Ok(())
 }
 
-pub fn delete_log_file() -> io::Result<()> {
+pub fn _delete_log_file() -> io::Result<()> {
```

Comment:
The functions in this file are general utility functions. Can we add exceptions for them so that we don't have to prefix them with underscores? It would just be confusing and annoying while debugging :)